Core/include/Acts/AmbiguityResolution/ScoreBasedAmbiguityResolution.hpp

```diff
@@ -80,11 +81,17 @@ class ScoreBasedAmbiguityResolution {
     std::size_t nSharedHits = 0;
   };
 
-  /// @brief MeasurementInfo : contains the measurement ID and the detector ID
-  struct MeasurementInfo {
-    std::size_t iMeasurement = 0;
-    std::size_t detectorId = 0;
-    bool isOutlier = false;
+  enum class TrackStateTypes : int {
```

Comment:
I suppose se can also go for `std::uint8_t`?

Reply:
yes uint_8  should work